src/styles/main.scss

```diff
@@ -0,0 +1,58 @@
+body {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vmin;
+}
+
+.empty-circle {
+  width: 80vmin;
+  height: 80vmin;
+  border: 2px dotted black;
+  border-radius: 50%;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.centre-circle {
+  position: absolute;
+  background-color: orange;
+  height: 5vmin;
+  width: 5vmin;
+  border-radius: 50%;
+  z-index: 2;
+}
+
+.minutes {
+  position: absolute;
+  background-color: blue;
```

Comment:
Consider using a more descriptive color name or hexadecimal color code for better readability and maintainability.